src/Engine/GameState/I_GameState.h

```diff
@@ -10,9 +10,26 @@ namespace gamestates {
     };
 }
 
+/**
+ * Many objects are intended to inherit I_GameState.
```

Comment:
Oh cool. This comment actually made it into the top level description:
![image](https://user-images.githubusercontent.com/12277640/33590171-edb984cc-d93a-11e7-89dd-758f281e9867.png)